modules/cvv/test/test_main.cpp

```diff
@@ -1,3 +1,3 @@
 #include "test_precomp.hpp"
 
-CV_TEST_MAIN(".")
+CV_TEST_MAIN(".", /*Empty VA_ARGS for C++11*/)
```

Comment:
`cvv` module is compiled with C++11 flags, and gcc does not like empty VA_ARGS in variadic macros. I hope it will not break anything.